CMakeLists.txt

```diff
@@ -1459,7 +1466,7 @@ foreach(group ${OC_C4GROUPS})
 			DEPENDS "${native_c4group}"
 			VERBATIM
 		)
-		install(FILES ${CMAKE_CURRENT_BINARY_DIR}/${group} DESTINATION share/games/openclonk)
+		install(FILES ${CMAKE_CURRENT_BINARY_DIR}/${group} DESTINATION "${OC_SYSTEM_DATA_DIR}")
```

Comment:
Sorry for the late response, but this doesn't work on Windows (or rather, all non-UNIX platforms, of which there is only one we support) because OC_SYSTEM_DATA_DIR is unset, which makes CMake unhappy.